### PR TITLE
[ko] Translate docs/home/supported-doc-versions documentation

### PR DESCRIPTION
**Pull request by anubha-v-ardhan**
Translated "This website contains documentation for the current version of Kubernetes and the four previous versions of Kubernetes." of https://github.com/kubernetes/website/blob/dev-1.21-ko.5/content/en/docs/home/supported-doc-versions.md in Korean which was missing earlier.

Issue -> https://github.com/kubernetes/website/issues/28512

### CODE REVIEW

File: content/ko/docs/home/supported-doc-versions.md [hunk 1]
```diff
@@ -7,3 +7,6 @@ card:
   weight: 10
   title: 사용 가능한 문서 버전
 ---
+
+이 웹 사이트에는 현재 버전의 Kubernetes
+에 대한 문서가 포함되어 있습니다 그리고 네 가지 이전 버전의 Kubernetes.
```
Comment (jihoon-seo) [inline content/ko/docs/home/supported-doc-versions.md:12]: [영어 원문]
> This website contains documentation for the current version of Kubernetes
and the four previous versions of Kubernetes.

```suggestion
이 웹사이트에서는 쿠버네티스 최신 버전 및 
이전 4개 버전에 대한 문서를 제공하고 있다.
```

**Comment by seokho-son** [comment]
@anubha-v-ardhan Thank you for the contribution!
/lgtm

Please squash commits in PR according to the contribution guideline :)
Thank you !
https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

**Comment by anubha-v-ardhan** [comment]
@seokho-son I think I did something wrong. I fetched and merged `kubernetes:main` to my `anubha-v-ardhan:patch- 2`. What should I do now

**Comment by seokho-son** [comment]
@anubha-v-ardhan 
Since base branch for feature-branch of this PR is `dev-1.21-ko.5`, 
merging feature-branch with `main` branch mixes commits in `dev-1.21-ko.5`, `main`(63 commits), and your commit(1 commit).

For squashing, you just needs to rebase in your local branch. no need to merge with other branch :)

I think you can cherry-pick your commit to resolve commit conflicts, 
or it is ok to close and open new PR according to your preference :)

**Comment by anubha-v-ardhan** [comment]
Thanks for the help @seokho-son. I am closing this and opening a new PR. This was my first ever PR in open source, sorry for the inconvenience caused. :' )

**Pull request closed without merging**